server/index.js

```diff
@@ -187,38 +189,36 @@ Visit the documentation for more information: https://bpanel.org/docs/configurat
     return clientsMap;
   }, new Map());
 
-  // Init bsock socket server
-  const socketHttpServer = http.createServer();
-  bsock.attach(socketHttpServer);
-
   // Init app express server
   const app = express.Router();
```

Comment:
It would be nice to get rid of express if we migrated over our other routes

Reply:
Yeah, I agree. I was actually looking into this originally but one of the advantages with express that we would lose is that there is a pretty robust middleware ecosystem. JJ reproduced a lot of the most important ones (bodyParser, cookies, etc.) but still nice to have that available. Using this new `bweb` server architecture might make that less important though. Also we should consider how we're going to do the backend plugin system as that will probably be leveraging middleware as well.